src/request.js

```diff
@@ -0,0 +1,20 @@
+const getScores = async (id) => { //eslint-disable-line no-unused-vars
+    const url = "https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/ubc1A0KC5FEKLnzaXFow/scores";
+    const response = await fetch(url, { method: "GET" });
+    const scores = await response.json();
+    return scores;
+  };
```

Comment:
- [ ] I see you are not using the id parameter, it is best practice to remove it. I removed it and it still works fine.